pghoard/monitoring/pushgateway.py

```diff
@@ -0,0 +1,48 @@
+"""
+Prometheus Pushgateway client
+
+"""
+import requests
+import socket
+
+
+class PushgatewayClient(object):
```

Comment:
No need to inherit from object on Python 3.x anymore.

Reply:
I didn't know about it, thanks for the review!

Reply:
Not strictly BTW needed for this PR but once these are all in can you please add a NEWS file entry for these changes.